build_test.sh

```diff
@@ -1,14 +0,0 @@
-#!/bin/bash
```

Comment:
I think it was very useful that one could run all the tests locally without dependening on CI infrastructure. It would be good to have build_test.sh back which builds everything locally.

Reply:
Ok, I added a Makefile to do just that. Now you can build and run the whole test suite with `make test`.